votepy/ordinal_election.py

```diff
@@ -0,0 +1,57 @@
+from typing import Iterable, Sequence
+
+
+class OrdinalBallot(list):
+    def __init__(self, ordering: Iterable, mapping=None):
```

Comment:
What is the type of `mapping`?

Reply:
Missed that, it's the same as in 'update_mapping'. Fixed